docs/extend/modular-resources/examples/csi.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Add a CSI Camera as a Modular Resource"
+linkTitle: "CSI Camera"
+weight: 40
+type: "docs"
+description: "How to add a CSI Camera as a modular resource of your robot."
+tags: ["board", "csi", "jetson", "serial", "module", "modular resources", "Python", "python SDK", "nvidia", "jetson orin", "jetson orin nano", "nano", "camera"]
+# SMEs: Sean
+---
+
+
+Viam provides a modular resource [extending](/extend/modular-resources/) the [camera API](/components/camera/#api) as a new `viam:camera:csi` model of [camera](/components/camera/).
```

Comment:
I thin kyou want to provide the value before the explanation - what does this resource allow readers to do?

Reply:
Ok done I think. Sentence is a little bare though-- maybe revisit this feedback tmrw

Reply:
(I will revisit)

Reply:
Added more talking about what I think is the value-add of CSI cameras and explaining more about what they are-- lmk what you think. Using info from here but not sure whether or not to link @npentrel  https://embeddedcomputing.com/technology/analog-and-power/mipi-csi-2-v3-0-enhances-imaging-reduces-cabling-for-embedded-systems-designers

Reply:
This is good but I think what's missing is why I'd need to use this over the normal camera component - see suggestion.

Reply:
This sounds good. In the context of Jetson devices one of the major advantages is using the hardware-accelerated [ISP](https://en.wikipedia.org/wiki/Image_processor) (image signal processing) library on the host machine called [libargus](https://docs.nvidia.com/jetson/l4t-multimedia/group__LibargusAPI.html).

Also, mechanically reliable connection from the camera to the host machine on mobile robotics platforms. (As opposed to using usb cables/ports).